### ISSUE TITLE
No emoji button anymore

### ISSUE THREAD

**Issue by Esther-Marieke** [mganss/EmojiAddIn#43]
### Application and version where the issue occurs (e.g. Thunderbird 52.0, Outlook Desktop 2016, Outlook Web)
Thunderbird 78.8.1 32bits
### Expected behavior
Emoji add-on is available
### Actual behavior
Emoji add-on is gone, the button is not in the toolbar anymore and is not visible in the menu where I should add it to the toolbar.
When I try to download the add-on, from the Thunderbird add-on page, again I get the message, the add-on is not compatible with my version of Thunderbird.
If I download the add-on anyway, and try to open it, I can not open a xpi file and Thunderbird is not available at the App's list I can try to open the xpi file with. (Firefox is)
I am just a user that is used to work with Thunderbird in Dutch, so please be simple in explanations on what to do next.

**Comment by mganss** [comment]
Is the add-on listed and activated in main menu -> Add-ons?

**Comment by Esther-Marieke** [comment]
If I looked at that menu-list it was not there, so I choose more add-ons, then I got the 'add-ons for Thunderbird' page. I did not see emoji in the list so I googled 'emoji thunderbird'. Again I got the Thunderbird Add-ons page and searched for emoji, all options were not compatible with this version of Thunderbird. I downloaded the add-on anyway but it did not find thunderbird to open it with. 

Today I tried again and now I can add the emoji add-on and the page says it is compatible with the version. The page looks different than yesterday so something happened there. So problem is solved but what is restored in the mean-time, no idea.....

**Comment by mganss** [comment]
@Esther-Marieke Does it work now?

**Comment by Esther-Marieke** [comment]
Well, not really. When I wrote to you I added the emoji add-on and saw it in Thunderbird. I opened it, saw it and closed the mail. When I now, at this moment, make a new concept e-mail, the button is not showing. When I look at the add-on list in the menu, it is there and activated. But it is not showing in the e-mail as a button. When I want to add it to the toolbar, it is not there. So no, problem not solved yet, sorry...

**Comment by Esther-Marieke** [comment]
I just found it via the top menu, so I can open it. But no button found yet.

**Comment by mganss** [comment]
Yes, the JoyPixels emoji set is this add-on. So this means you can open the sidebar and show the list of emojis?
If that's the case and you want to add the Emoji button (to enable/disable the sidebar) to the toolbar, please follow these steps: http://kb.mozillazine.org/Thunderbird_:_Tips_:_Customize_Toolbar

**Comment by Esther-Marieke** [comment]
It now is added to the toolbar and stays there if I open and close new concepts, so it seems to be resolved.
I don't know if you can do something about it but the hyperlink  Customize Toolbar dialog in http://kb.mozillazine.org/Thunderbird_:_Tips_:_Customize_Toolbar gives a 404 Error. 
Thanks for your help!

**Comment by mganss** [comment]
This is more up to date: https://support.mozilla.org/en-US/kb/how-customize-toolbars

**Comment by Esther-Marieke** [comment]
Yes, that one is much better!
